scripts/CEE/get-rhoc-connector-info/get-rhoc-connector-info.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+timestamp=$(date +%s)
+outputDir="inspect.${connector_id}.${timestamp}"
+
+mkdir "${outputDir}"
+
+set -o pipefail
+set -o nounset
+
+echo "Collecting resources for connector ${connector_id}"         
+
+oc get \
+    ManagedConnector,KameletBinding,Integration,IntegrationKit,KafkaConnect,KafkaConnector,Deployment,ReplicaSet,Pod,PodDisruptionBudget,NetworkPolicy,ConfigMap,Service,EndpointSlice \
+    -l cos.bf2.org/connector.id="${connector_id}" \
+    --all-namespaces \
+    -o yaml > ${outputDir}/resources.yaml
```

Comment:
Can be put in double quote to prevent globbing and word splitting.
Build is also showing this in error

Reply:
done